docs/section10/10-7.md

```diff
@@ -0,0 +1,121 @@
+- Get dictionary list of baseline zones and the associated HVAC systems: `hvac_zone_list_w_area_dict_b = get_hvac_zone_list_w_area (B_RMR)`  
+    - For each hvac system in the the B_RMR: `for hvac_b in B_RMR...HeatingVentilationAirConditioningSystem:`
+        - Reset boolean variable that indicates if the hvac system is type 3 or 4 (used in rule assertion): `sys_type_3_or_4 = FALSE`
+        - Reset boolean variable that indicates if the hvac system is in the smallest capacity bracket (used in rule assertion): `smallcapBracket = FALSE`
+        - Get the hvac system type for hvac_b: `hvac_sys_type_b = baseline_hvac_sys_type_ids_dict_b.keys()[list(baseline_hvac_sys_type_ids_dict_b.values()).index(hvac_b.id)]`
+        - Get list of zones that the HVAC system serves: `zone_list_b = hvac_zone_list_w_area_dict_b[hvac_b.id]["ZONE_LIST"]`  
+        - Check if HVAC system is of type 1, 1b, 2, 3, 3b, 4, 5, 5b, 6, or 6b AND that is does not serve more than one floor, if meets criteria then carry on: `if hvac_sys_type_b in ["SYS-1", "SYS-1b", "SYS-2", "SYS-3", "SYS-3b", "SYS-4","SYS-5", "SYS-5b", "SYS-6", "SYS-6b"] and hvac_b.id not in hvac_sys_serve_more_than_one_flr_list_b:` 
+            - Reset # zones defined for all zones boolean variable to TRUE: `num_zones_defined_for_all_zones = TRUE`
+            - Reset number of zones count: `num_zones = 0`
+            - Check if the # of zones associated with a thermal block (i.e., zone) is defined for each zone: `For zone_b in zone_list_b:`
+                - if zone_b.zones does not equal null then get the number of zones: `if zone_b.zones <> Null:`
+                    - Get number of zones and add to running list: `num_zones = num_zones + zone_b.zones`
+                - Else, set num_zones_defined_for_all_zones to false: `num_zones_defined_for_all_zones = FALSE`
+            Loop through each and get the attributes of the cooling system(s) associated with this HVAC (e.g. efficiency, capacity, etc.)
+            - For each cooling_system_b in hvac_b: `for cooling_system_b in hvac_b.cooling_system:`
+                - Check if rated cooling capacity does not equal null: `if cooling_system_b.rated_total_cool_capacity <> Null:`
+                    - Set cooling capacity equal to the rated cooling capacity: `total_cool_capacity_b = cooling_system_b.rated_total_cool_capacity`
+                - Else, rated equals Null then set the cooling capacity equal to the design cooling capacity:`total_cool_capacity_b = cooling_system_b.design_total_cool_capacity`
+                - Get the full load no fan cooling efficiency: `full_load_efficiency_b = cooling_system_b.full_load_efficiency_no_fan`
+                - Reset meets size category or lower size category efficiency boolean variable: `meets_lower_or_equiv_size_category_eff_b = FALSE` 
```

Comment:
We now have zone.equipment_count field for this. The exact name of the data element is still in flux, but I would use this one as a placeholder for now.